circleci/README.md

```diff
@@ -0,0 +1,36 @@
+# Circle CI
+ 
+### Install
+ 
+Running the below command to install the whole CircleCI command set.
+ 
+```
+/nc csm_install circleci
+```
+ 
+### Prerequisite:
+To be able to run the commands below, you need to create 3 secrets.
+ 
+These are :
+`projectToken` - CircleCI allows you to create a Token on a project basis to access API, Make sure you provide the token here which can access all those branches which you want to use via this command.
```

Comment:
Make this shorter, simply: This is your CircleCI API access token. 

Can we provide a link where this can be generated in the circleci ui?